tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/UpgradeAssistantSettingsUtil.java

```diff
@@ -85,6 +85,22 @@ public boolean accept( File dir, String name )
         return null;
     }
 
+    public static <T> T getObjectFromStore( Class<T> clazz, String suffix ) throws IOException
```

Comment:
Two problems
1) there i only one caller to this method, MigrationUtil.updateProblemToStore(), no other callers are looking for the "suffixed" json files, why not?
2) we shouldn't be using suffixes, because that makes the data more complicated.  We should just be encoding the "project" information into the java class itself, so that we are always only serializing the entire class to json file.